native/tools/swig-config/scene.i

```diff
@@ -281,7 +281,7 @@ using namespace cc;
 %attribute_writeonly(cc::Node, Mat4&, matrix, setMatrix);
 %attribute(cc::Node, uint32_t, hasChangedFlags, getChangedFlags, setChangedFlags);
 %attribute(cc::Node, bool, _persistNode, isPersistNode, setPersistNode);
-%attribute(cc::Node, cc::MobilityMode, mobility, getMobility, setMobility);
+%attribute(cc::Node, cc::MobilityMode, _mobility, getMobility, setMobility);
```

Comment:
`_mobility` is public field which will automatically bound.
No need to add %attribute for `_mobility` here.

Reply:
ok